.github/workflows/cflite_pr.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       matrix:
         sanitizer: [address, undefined, memory]
     steps:
+    - run: sudo sysctl -w vm.mmap_rnd_bits=28
```

Comment:
Could you add a short comment about this?

Reply:
OK, then let's merge this as is.

Reply:
I opened https://github.com/systemd/systemd/pull/31803